spec/commands/outdated_spec.rb

```diff
@@ -72,7 +72,7 @@
   end
 
   describe "with --group option" do
-    def test_group_option(group = nil, gems_list_size = 1)
+    before :each do
```

Comment:
We're not consistent here, but the preferred style should be `before do` instead of `before :each do`

Reply:
I changed all the `before :each do` occurrences from this spec to be consistent.